index.html

```diff
@@ -45,7 +42,33 @@ <h1 class="heading__text">
         <button type="submit">Let's Drink</button>
       </form>
       <section class="drinks">
+        <!-- HTML Validator recommends adding a heading here. Something to consider. -->
```

Comment:
what kind of heading are we talking about?

Reply:
An h2? I don't think we _need_ one, but I passed our HTML through a validator and it recommended one.